Shoko.Server/Providers/AniDB/HTTP/HttpAnimeParser.cs

```diff
@@ -473,20 +473,26 @@ private static ResponseTag ParseTag(int animeID, XmlNode node)
         }
 
         var tagDescription = TryGetProperty(node, "description")?.Replace('`', '\'');
+        int.TryParse(TryGetAttribute(node, "parentid"), out var parentTagID);
         int.TryParse(TryGetAttribute(node, "weight"), out var weight);
+        bool.TryParse(TryGetAttribute(node, "verified"), out var verified);
         bool.TryParse(TryGetAttribute(node, "localspoiler"), out var lsp);
         bool.TryParse(TryGetAttribute(node, "globalspoiler"), out var gsp);
+        if (!DateTime.TryParse(TryGetAttribute(node, "update"), out var lastUpdated))
+            lastUpdated = DateTime.UnixEpoch;
 
         return new ResponseTag
         {
             AnimeID = animeID,
             TagID = tagID,
+            ParentTagID = parentTagID > 0 ? parentTagID : null,
             TagName = tagName,
             TagDescription = tagDescription,
             Weight = weight,
+            Verified = verified,
             LocalSpoiler = lsp,
             GlobalSpoiler = gsp,
-            Spoiler = lsp || gsp
```

Comment:
Are you just removing the general spoiler field?

Reply:
It was never used.

Reply:
Also, if it's a global spoiler, then it's a spoiler. The local spoiler is only for when the tag is used on that particular anime.

Reply:
> It was never used.

Outside the setters i mean.

Reply:
That's fine. I was just asking for clarification